test/regressions/fixtures/LinkJoy/LinkCard.js

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import Avatar from '@mui/joy/Avatar';
+import Box from '@mui/joy/Box';
+import Card from '@mui/joy/Card';
+import Link, { linkClasses } from '@mui/joy/Link';
+import Typography from '@mui/joy/Typography';
+
+export default function LinkCard() {
+  return (
+    <Box sx={{ p: 2 }}>
+      <Card variant="outlined" sx={{ display: 'flex', gap: 2 }}>
+        <Avatar size="lg" src="/static/images/avatar/1.jpg" />
+        <Link
+          overlay
+          href="#introduction"
```

Comment:
I wonder why the link's `href` is` #introduction`—Doesn't it make more sense to be `#overlay-prop`?

Reply:
agree!

Reply:
Oh, in this case, it does not matter. It's just a visual regression test, any `#…` works fine.